CHANGELOG.md

```diff
@@ -45,7 +46,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Initial release
 - Use https://github.com/cyberark/conjur-api-go to read configuration.
 
-[Unreleased]: https://github.com/cyberark/terraform-provider-conjur/compare/v0.4.0...HEAD
+[Unreleased]: https://github.com/cyberark/terraform-provider-conjur/compare/v0.5.0...HEAD
+[0.5.0]: https://github.com/cyberark/terraform-provider-conjur/compare/v0.4.0...v0.5.0
 [0.4.0]: https://github.com/cyberark/terraform-provider-conjur/compare/v0.3.0...v0.4.0
```

Comment:
This is outside of your change, but this line is using the wrong compare. It should reference `v0.3.1` instead of `v0.3.0`:
```
[0.4.0]: https://github.com/cyberark/terraform-provider-conjur/compare/v0.3.1...v0.4.0
```